tests/aediis-2/input.dat

```diff
@@ -0,0 +1,18 @@
+# EDIIS test case from 10.1063/1.1470195
```

Comment:
```suggestion
#! EDIIS test case from 10.1063/1.1470195
```
(since you took the trouble to clean up the missing comments for docs last week :-)